library/network/src/modules/SuSEFirewall.rb

```diff
@@ -405,6 +414,64 @@ def _sf2_to_firewalld_service(service)
       end
     end
 
+    # Function for getting exported SuSEFirewall configuration
+    #
+    # @return	[Hash{String => Object}] with configuration
+    def Export
+      deep_copy(@SETTINGS)
+    end
+
+    # Function for setting SuSEFirewall configuration from input
+    #
+    # @param	map <string, any> with configuration
+    def Import(import_settings)
+      Read()
+      import_settings = deep_copy(import_settings)
+      # Sanitize it
+      import_settings.keys.each do |k|
+        if !GetKnownFirewallZones().include?(k) and !@@key_settings.include?(k)
+          Builtins.y2warning("Removing invalid key: %1 from imported settings", k)
```

Comment:
So the AutoYaST configuration is not compatible. That is probably OK but we should make sure that the users expect that. 
See also:
- [the firewall part of the AY schema](https://github.com/yast/yast-firewall/blob/master/src/autoyast-rnc/firewall.rnc)
- the [SLES AY docs](https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html) ... does not mention autoconfiguring the firewall at all :laughing: .